quotes/QuoteModel.swift

```diff
@@ -19,10 +19,10 @@ import Foundation
     "authorSlug" : "henri-poincare",
     "author" : "Henri Poincaré"
  }
-*/
+ */
 
 struct Quote: Codable {
-    let length:Int
+    let length: Int
     let id, content: String
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/swift-linting-formatting-workshop/blob/6bd6a61968c868f71f3fef73fa1bffcb3d3a399c/quotes/QuoteModel.swift#L26"></span>Variable name should be between 3 and 40 characters long: 'id'<br />
<code>identifier_name</code> <code>QuoteModel.swift:26</code></td>
    </tr>
  </tbody>
</table>